src/ethereum/EnigmaContractReaderAPI.js

```diff
@@ -494,7 +494,7 @@ class EnigmaContractReaderAPI {
        * */
       [constants.RAW_ETHEREUM_EVENTS.ReceiptVerified]: event => {
         return {
-          taskId: nodeUtils.remove0x(event.returnValues.bytes32s[1]),
+          taskId: nodeUtils.remove0x(event.returnValues.taskId),
```

Comment:
Please update the the `SecretContractDeployed` event as well (for alignment):
```
[constants.RAW_ETHEREUM_EVENTS.SecretContractDeployed]: event => {
        return {
          secretContractAddress: nodeUtils.remove0x(event.returnValues.bytes32s[0]),
          codeHash: event.returnValues.bytes32s[2],
          stateDeltaHash: event.returnValues.bytes32s[3]
        };
      },
```
to:
```
[constants.RAW_ETHEREUM_EVENTS.SecretContractDeployed]: event => {
        return {
          secretContractAddress: nodeUtils.remove0x(event.returnValues.taskId),
          codeHash: event.returnValues.bytes32s[2],
          stateDeltaHash: event.returnValues.bytes32s[3]
        };
      },
```